notebook/templates/page.html

```diff
@@ -38,13 +38,10 @@
             moment: 'components/moment/moment',
             codemirror: 'components/codemirror',
             termjs: 'components/term.js/src/term',
-            typeahead: 'components/jquery-typeahead/dist/jquery.typeahead'
+            marked: 'components/marked/marked.min',
+            caja: 'components/google-caja/html-css-sanitizer-minified'
```

Comment:
Only via node.  Since @Carreau added that recently (1-2 PRs ago), I didn't see any reason to load it into bower for backwards compat.